internal/review.go

```diff
@@ -0,0 +1,195 @@
+package internal
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"slices"
+	"strings"
+
+	"github.com/mitchellh/colorstring"
+	"github.com/spf13/cobra"
+
+	"github.com/nobl9/nobl9-go/sdk"
+)
+
+var (
+	errReviewTooManyArgs    = errors.New("'review set' command accepts only single SLO name as an argument")
+	errReviewInvalidOptions = errors.New("you must provide the SLO name as an argument")
+)
+
+type ReviewCmd struct {
+	client  *sdk.Client
+	project string
+	status  string
+	note    string
+	sloName string
+}
+
+//go:embed review_example.sh
+var reviewExample string
+
+// NewReviewCmd returns cobra command review with all its flags and subcommands.
+func (r *RootCmd) NewReviewCmd() *cobra.Command {
+	review := &ReviewCmd{}
+
+	cmd := &cobra.Command{
+		Use:   "review",
+		Short: "Manage SLO review (Enterprise Edition only)",
+		Long: `Manage SLO review.
+
+Note: This feature is only available in Enterprise Edition tier.`,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			review.client = r.GetClient()
+		},
+	}
+
+	cmd.AddCommand(review.NewSetCmd())
+
+	return cmd
+}
+
+// NewSetCmd creates the set subcommand for reviews
+func (r *ReviewCmd) NewSetCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set <SLO_NAME>",
```

Comment:
how about `submit`? feels more natural, even the skipped status can have a comment and you're still submitting something there

```suggestion
		Use:   "submit <SLO_NAME>",
```